docs/source/includes/_authentication.md

```diff
@@ -3,5 +3,138 @@
 Panoptes API provides an OAuth 2.0 authentication mechanism. (TODO: Describe
 Authentication Workflow).
 
+## How do I authenticate with the Zooniverse API?
 
+Some requests to the Zooniverse API will respond without supplied authentication details as they are public but most require an authentication token to be supplied to access our systems as a known user.
 
+This authentication token is known as a bearer token and is usually supplied as a HTTP `Authorization` header with the value prefixed by `Bearer` and then the token data. For example:
+
+`Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c`
+
+The bearer tokens we produce are actually [JWT tokens](https://jwt.io/) in and of themselves. This means that if you have a bearer token, you can unpack it, verify that it was signed by our servers, and find your user ID, your permissions and some other things.
+
+## Tools to interact with the API
+
+You can use Postman, Paw, cURL or one of our official Zooniverse libraries to authenticate and create requests to the API:
+
++ Python
+  + https://github.com/zooniverse/panoptes-python-client
++ JavaScript
+  + https://github.com/zooniverse/panoptes-javascript-client
++ Ruby
+  + https://github.com/zooniverse/panoptes-client.rb
+
+
+## Example using Postman
+
+[Postman](https://www.getpostman.com/) is a graphical tool to help you make API calls.
+
+### How do you get a token?
+
+```http
+POST /oauth/token HTTP/1.1
+Content-Type: application/json
+
+{
+  "grant_type": "client_credentials",
+  "client_id": "YOUR_CLIENT_ID",
+  "client_secret": "YOUR_CLIENT_SECRET"
+}
+```
+
+```ruby
```

Comment:
@adammcmaster would you happen to have a python-lib-using snippet on your machine that prints a token to stdout?

Reply:
```suggestion
```python
from panoptes_client import Panoptes
client = Panoptes.connect(username='', password='')
print(client.get_bearer_token())
```

Reply:
heh, I didn't really mean to make that a suggested change to this line.